.github/workflows/check_unit_tests.yml

```diff
@@ -81,6 +81,9 @@ jobs:
 
         python3 buildscripts/ci/linux/tools/lcov_badger.py filtered_coverage.info coverage_badge.svg
 
+    - name: Push to S3
+      if: github.repository == 'musescore/MuseScore'
```

Comment:
This step is now potentially also run when `github.event_name == 'schedule' || inputs.code_coverage == 'true'` is false

Reply:
Yes, sloppy me. Fixed it.